e2e/runner/run_cypress_local.ts

```diff
@@ -26,6 +28,7 @@ const derivedOptions = {
   CYPRESS_ALL_FEATURES_TOKEN: userOptions.ENTERPRISE_TOKEN,
   QA_DB_ENABLED: userOptions.START_CONTAINERS,
   BUILD_JAR: userOptions.BACKEND_PORT === 4000,
+  START_BACKEND: userOptions.BACKEND_PORT === 4000,
```

Comment:
I'm actually confused this `BACKEND_PORT` with
https://github.com/metabase/metabase/blob/a71ceea29649d52a1e6324f1ca04473edc327b7b/e2e/runner/sample-apps-shared/constants/sample-app-setup-configs.js#L7

I guess the 4500 is meant to be the sample app backend, not Metabase backend for Cypress, correct? If that's the cas,e can we be more clear about which backend is for which purpose?

If that's the case I'd personally name this new option
```suggestion
  START_CYPRESS_BACKEND: userOptions.BACKEND_PORT === 4000,
```

or something.

Reply:
This is the port that you linked: the port of auth BE of a sample app

Reply:
`START_METABASE`?

Reply:
Or you could name the new `BACKEND_PORT`, `SAMPLE_APP_BACKEND_PORT` wdyt?

Reply:
I just want them to look different, so I know they're not the same backend.

Reply:
> Or you could name the new BACKEND_PORT, SAMPLE_APP_BACKEND_PORT wdyt?
The env variables in `e2e/runner/sample-apps-shared/constants/sample-app-setup-configs.js` are passed directly to sample apps. It means that we should have `SAMPLE_APP_BACKEND_PORT` env variable inside Sample App repo, at least in its docker-compose file. I don't like it.

Initially I had `AUTH_PROVIDER_PORT` instead of `BACKEND_PORT` but it conflicts with the documentation here:
https://www.metabase.com/docs/latest/embedding/sdk/quickstart-with-sample-app#set-up-the-application-environment

Reply:
ok that makes sense.

Reply:
We can use `AUTH_PROVIDER_PORT` but only for .env.docker file, and `BACKEND_PORT` in a regular `.env` file to match with the documentation page

Reply:
let's change the main Metabase env name then, it looks simpler that way. If you need the PR you can merge now, and follow up with Ryan later to see if he's okay with changing the main env name.

Reply:
Actually we can really change BACKEND_PORT to AUTH_PROVIDER_PORT and update documentation, because right now we have `VITE_AUTH_PROVIDER_URI` but `BACKEND_PORT`

Reply:
I added a commit https://github.com/metabase/metabase/pull/54250/commits/3e65e9c0c811976331d9e766f231a3788648e80e

And will sync documentation to the `metabase.github.io` as soon as I merge this PR